types/multipath.pp

```diff
@@ -1 +1,3 @@
+# @ summary Puppet-Type for Elasticsearch's Multipath
```

Comment:
```suggestion
# @summary Puppet-Type for Elasticsearch's Multipath
```

Reply:
you added this change, but the change isn't correct :D

Reply:
I reverted the non-relevant Changes.    But the following also happens in `master`:  

```
[warn]: Missing documentation for Puppet type alias 'Elasticsearch::Status' at types/status.pp:1.                                                                                                                    
[warn]: Missing documentation for Puppet type alias 'Elasticsearch::Multipath' at types/multipath.pp:1.

[…]

  1) Puppet::Type::Elasticsearch_keystore::ProviderElasticsearch_keystore instances with one instance                                                                                                                
     Failure/Error: stdout = execute([command(:keystore)] + args, options)                                                                                                                                           
                                                                                                                                                                                                                     
       #<Puppet::Type::Elasticsearch_keystore::ProviderElasticsearch_keystore (class)> received :execute with unexpected arguments                                                                                   
         expected: (["/usr/share/elasticsearch/bin/elasticsearch-keystore", "list"], {:custom_environment=>{"ES_INCLUDE"=>"/etc/default/elasticsearch-es-01", "ES_PATH_CONF"=>"/etc/elasticsearch/es-01"}, :failonfai
l=>true, :gid=>"elasticsearch", :uid=>"elasticsearch"})                                                                                                                                                              
              got: (["/usr/share/elasticsearch/bin/elasticsearch-keystore", "list"], {:custom_environment=>{"ES_INCLUDE"=>"/etc/sysconfig/elasticsearch-es-01", "ES_PATH_CONF"=>"/etc/elasticsearch/es-01"}, :failonf
ail=>true, :gid=>"elasticsearch", :uid=>"elasticsearch"})                                                                                                                                                            
       Diff:                                                                                                                                                                                                         
       @@ -1,6 +1,6 @@                                                                                                                                                                                               
        [["/usr/share/elasticsearch/bin/elasticsearch-keystore", "list"],                                                                                                                                            
         {:custom_environment=>                                                                                                                                                                                      
       -   {"ES_INCLUDE"=>"/etc/default/elasticsearch-es-01",                                                                                                                                                        
       +   {"ES_INCLUDE"=>"/etc/sysconfig/elasticsearch-es-01",                                                                                                                                                      
            "ES_PATH_CONF"=>"/etc/elasticsearch/es-01"},                                                                                                                                                             
          :failonfail=>true,                                                                                                                                                                                         
          :gid=>"elasticsearch",                                                                                                                                                                                     
```